src/Compilers/CSharp/Portable/CommandLine/CSharpCompiler.cs

```diff
@@ -254,6 +254,15 @@ public override void PrintLogo(TextWriter consoleOutput)
             consoleOutput.WriteLine();
         }
 
+        /// <summary>
+        /// Print compiler version
+        /// </summary>
+        /// <param name="consoleOutput"></param>
+        public override void PrintVersion(TextWriter consoleOutput)
+        {
+            consoleOutput.WriteLine(GetAssemblyFileVersion());
+        }
+
```

Comment:
GetAssemblyFileVersion is an existing non-virtual method with a number of use sites. It does not do the same thing as required for the /version flag (it reports the version of the common compiler assembly). I'd rather not create a third assembly version API in CommonCompiler to facilitate that sharing, so I'm not going to do that.

---
In reply to: [79923998](https://github.com/dotnet/roslyn/pull/13955#discussion_r79923998) [](ancestors = 79923998)